cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -186,6 +109,47 @@ const GraphView: FC = () => {
         formIsDirty
     );
 
+    if (isLoading) {
+        return (
+            <Box sx={{ position: 'relative', height: '100%', width: '100%', overflow: 'hidden' }} data-testid='explore'>
+                <GraphProgress loading={isLoading} />
+            </Box>
+        );
+    }
+
+    const dataCollectionLink = (
+        <Link
+            target='_blank'
+            href={'https://support.bloodhoundenterprise.io/hc/en-us/sections/17274904083483-BloodHound-CE-Collection'}>
+            Data Collection
+        </Link>
+    );
+
+    const fileIngestLink = (
+        <Link component={RouterLink} to={ROUTE_ADMINISTRATION_FILE_INGEST}>
+            File Ingest
+        </Link>
+    );
+
+    if (isError) throw new Error();
+
+    if (!data.length)
+        return (
+            <Box position={'relative'} height={'100%'} width={'100%'} overflow={'hidden'}>
+                <NoDataAlert dataCollectionLink={dataCollectionLink} fileIngestLink={fileIngestLink} />
```

Comment:
The props for this component seem a little strange to me. I think I would prefer to see us just using the MUI <Alert> component here directly.

Reply:
They do look a bit strange only looking at the CE side but since the data collection link is different and we are showing some info conditionally based on whether there is a file ingest link they are effectively utilized in BHE.